src/core/GCAudio.qml

```diff
@@ -214,7 +214,10 @@ Item {
             source = ""
             gcaudio.done()
         } else {
-            source = ""
+            // on Ubuntu Touch, emptying the source result in an Audio error which triggers that method again endlessly
```

Comment:
should it be the same on line 140?

Reply:
idk, but haven't spot any audio issue with it. Need to dig a bit to know the difference between those two

Reply:
If I remember correctly, "play" is called to play the sound directly, while the other one is when you append audio to play after the current one has finished.

Intro audio and when you lose on an activity (for example reversecount) will play using "play". You can lose twice and it should be the same audio (try_again) playing twice in a row